### ISSUE TITLE
Tags does not work when inserted before an stereotype

### ISSUE THREAD

**Issue by JohnLeyva** [plantuml/plantuml#1162]
Objective is to have define a $tag that can be injected between stereotypes that can be used to remove items. Functions will be used so that's why the exact position can not be controlled.

Noticed that $tag after all stereotypes does work
$tag between stereotypes is just rendered
$tag before stereotypes generate an error

@startuml

!function $f () 
!return 'AAC ' + ' $all'
!endfunction

' Uncomment the following will generate an error
'node EE $all <<bb>> {
'}

node DD {
}

node CC $all {
}

node BB <<dd>> $all <<bb>> {
}
note left of BB
   Expected to be removed
end note

node $f() {
}

remove $all

@enduml

**Comment by arnaudroques** [comment]
Thanks for the feedback.
This [is fixed in last snapshot](http://www.plantuml.com/plantuml/uml/SoWkIImgAStDuShBJqbLS75HgEQgvO9md9qLL19pSf14d9mKR6pIylCDxUpGfHmTmI9mUOWKLr5gRdvPAbYAoo4rBmLe4m00).

**Comment by claudio-salvio** [comment]
@JohnLeyva 
I tried to reproduce the error you mentioned before the modification introduced by @arnaudroques to solve it.

I went to https://www.plantuml.com/plantuml/uml/ and pasted your example **without** uncommenting the offending lines.
I found that the code gives error on the line:
```plantuml
node BB <
```

Image with the error shown:
![image](https://user-images.githubusercontent.com/66575834/199579115-4d9bc58f-8305-4065-8a10-1258301bf723.png)

Is this what is expected to happen?

Best regards,
Claudio Salvio
